test/index.js

```diff
@@ -57,6 +57,20 @@ setupTest('twig-filters', {
   }
 })
 
+setupTest('namespaces', {
+  pluginOpts: {
+    'metalsmith-jstransformer': {
+      engineOptions: {
+        twig: {
+          namespaces: {
+            icons: '/images/icons/'
```

Comment:
Since we're exposing the `@icons` Twig namespace, I'd expect that we'd use `@icons` somewhere in a Twig file?